pyzeebe/credentials/typing.py

```diff
@@ -1,4 +1,7 @@
-from typing import Protocol, Tuple, Union
+from typing import Any, Protocol, Sequence, Tuple, Union
+
+ChannelArgumentType = Sequence[Tuple[str, Any]]
```

Comment:
Please, move it to https://github.com/camunda-community-hub/pyzeebe/blob/master/pyzeebe/types.py

Reply:
I will do that and update the documentation, then this should be ready to merge.